flake.nix

```diff
@@ -444,7 +444,7 @@
                 ];
               };
 
-              gateway-ui = pkgs.mkShell (shellCommonNative
+              web = pkgs.mkShell (shellCommonNative
```

Comment:
maybe rename the shell to `fedimint-ui`? we should make sure to update the docs that show how to launch and use this shell